lib/rest-adapter.js

```diff
@@ -256,12 +257,27 @@ RestAdapter.prototype.createHandler = function() {
 
   var handleUnknownPaths = this._shouldHandleUnknownPaths();
 
+  // mount rest route handler
+  var customRouter = new RestRouter();
+  root.use(customRouter.handle());
+  
   classes.forEach(function(restClass) {
     var router = express.Router();
     var className = restClass.sharedClass.name;
 
     debug('registering REST handler for class %j', className);
-
+    
+    // Mount the remoteClass router on all class routes.
+    restClass
+      .routes
+      .forEach(function (route) {
+        debug('    at %s', route.path);
+        root.use(route.path, router);
+        route.fullPath = restClass.getPath(); //add fullPath for Trie insertion
+        customRouter.registerPathAndHandlers(route);
+        //Do we need to register above class route?if yes, what's all method verb?
```

Comment:
> Do we need to register above class route?if yes, what's all method verb?

No, this does not make sense to me. There is no such thing as a class route. Only methods can contribute routes.